Integrations/Censys/Censys.py

```diff
@@ -1,61 +1,70 @@
 import demistomock as demisto
 from CommonServerPython import *
+
+''' IMPORTS '''
 import requests
 
+''' GLOBAL VARIABLES '''
+API_URL = demisto.params()['url']
+API_ID = demisto.params()['apiid']
+API_SECRET = demisto.params()['secret']
 USE_SSL = not demisto.params().get('insecure', False)
 
 
-def sendRequest(method, url, uri, user, passwd, data=None):
-    if method.lower() == 'post':
-        try:
-            json.loads(data)
-        except ValueError:
-            return '### Error: Data is not in JSON format'
+def test_module():
+    url_suffix = "view/ipv4/8.8.8.8"
+    res = requests.get(API_URL + url_suffix, auth=(API_ID, API_SECRET), verify=USE_SSL)
+    if res.status_code == 200:
+        demisto.results('ok')
+    else:
+        demisto.results('test failed')
```

Comment:
return the real response, this info will not give the user an informative enough problem